misc/dist/ios_xcode/godot_ios/godot.xcconfig

```diff
@@ -6,7 +6,7 @@
 CONFIGURATION_BUILD_DIR = "$(BUILD_DIR)/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)";
 SDKROOT = iphoneos;
 VALID_ARCHS = "armv7 armv7s arm64 x86_64";
-ENABLE_BITCODE = NO
+ENABLE_BITCODE = NO;
```

Comment:
Would be good if this hotfix was amended into the commit where the line was added in the first place.